pkg/monitoring/exporter_kafka_test.go

```diff
@@ -36,7 +36,7 @@ func TestKafkaExporter(t *testing.T) {
 		nodes := []NodeConfig{generateNodeConfig()}
 		exporter, err := factory.NewExporter(ExporterParams{chainConfig, feedConfig, nodes})
 		require.NoError(t, err)
-		envelope, err := generateEnvelope()
+		envelope, err := generateEnvelope(ctx)
```

Comment:
I'm confused by the introduction of context here. Most of these methods have no concurrency in their implementation - ctx is not needed and won't help. 

Take `generateEnvelope`, it produces a structure filled with random values. https://github.com/smartcontractkit/chainlink-common/pull/312/files#diff-be6ec1c2691cf18a8a73b7ea25c78bab3dc41d5bccde1df3e9626310e849306bR454-R477

Reply:
What is the relevance of concurrency? We are motivated by carrying request scoped values for tracing.

This func calls `generateContractConfig`, which calls `median.StandardOnchainConfigCodec.Encode`, which now requires a context.